### ISSUE TITLE
No Donate button on home page

### ISSUE THREAD

**Issue by stevenfeldman** [OSGeo/osgeo#43]
We currently have a donate button on the home page which allows people to make small donations via PayPal. We used this feature to crowdfund part of the of Travel Grant Programme.

Can we get that button recreated on the home page pls

**Comment by jodygarnett** [comment]
Marking this as content as we have the skills to do so. @stevenfeldman for the geoserver code sprint I provided the step by step instructions for using paypal to donate to a specific project or activity.  Would you like the donate button to go to such a page, or direct to paypal.

I am also wondering where such a button could go on the page without loosing focus (it literally on the subject of how visitors can help osgeo, rather than on how osgeo can help visitors).

**Comment by andreasneumann** [comment]
I agree with @stevenfeldman - currently it is really hard to find the PayPal address if someone wants to donate towards OSGeo.

Could the PayPal button or link be added again to the OSGeo webpage?

**Comment by jodygarnett** [comment]
Here is a request to SAC to provide a PayPal button: https://trac.osgeo.org/osgeo/ticket/2056 (which provides dropdown for donation categories).

We are getting some overlap with the sponsorship page, which also includes instructions for using PayPal.

* https://www.osgeo.org/about/how-to-become-a-sponsor/

I would like to try and do a better job of these two things so that fundraising does not distract from outreach mission of the website.

Would it be okay to have a donate button on the sponsorship page, project pages and initiatives pages? Basically the places where the public could show their support.

**Comment by stevenfeldman** [comment]
A quick scan of the web site suggests that multiple content creators are possibly a source of inconsistency in the way we do things.

For example - on the resources tab we have an enormous number of pdfs uploaded by geo for all, this is going to get out of control unless we offer better filtering and some archiving. The search and filter don’t seem to work for me but that might be a safari thing (don’t have time to test with another browser right now). Alternatively we could encourage them to host their newsletters on their web site and just provide one link in the resources section of the OSGeo web site

Initiatives seem to be proliferating (I acknowledge my own small bit of responsibility here) - that may be a great thing but should someone be deciding what constitutes an initiative that should appear on the web site? Maybe the marketing committee does do this?

Member bios are inconsistent, some do not have photos some do not have any content  apart from a couple of ticks in projects e.g. https://www.osgeo.org/member/pawley/ <https://www.osgeo.org/member/pawley/>. 

The member search also returns some unexpected results e.g. Jo Cook, Ian Turton and Matt Walker (friends and colleagues of mine) all have Astun Technology in their bios and under the ‘company’ field, I have Astun Technology in bio:
if you search for Astun only Jo’s name is returned
If you search for Astun Technology none of our names are returned

The service providers section shows 9 logos on the first page (with loads of follow on pages- it appears that these first 9 are not randomly selected, this could cause some upset. Filtering by location takes you to a nightmare dropdown which IMO needs cleaning up. Surely the location filter should be at country level or perhaps in the US at state but not at individual towns/cities where people are located?

There is a lot of news being posted which is great but it appears in 9 large blocks with 63 follow on pages! Perhaps we need to look at a way of presenting more news items on the front page and also a way of accessing past news by having a date filter or selector? Are there any criteria as to what constitutes OSGeo news (community and foundation) and who can post?

Nothing disastrous in these observations but perhaps an indication of how the site to become unwieldy and inconsistent in behaviour and content if we don’t try to set some policies or guidelines now.

Would offer to help but I am probably too dictatorial for such a role (and I am away for most of May and then back to the TGP)

Hope you are well, looking forward to catching up in Dar, or Guimaraes if you are coming to FOSS4G Europe

Cheers
______
Steven

**Comment by jodygarnett** [comment]
@stevenfeldman we should make a separate discussion on this, marketing committee has some support for 2018, and budget to address high priority issues (such as search possibly not extending to custom fields?).

**Comment by jodygarnett** [comment]
There is now *Donate* button on the home page - but I hate it. It is the very first thing we bring visitors attention to and it is not in keeping with the website values of *public service* / *public outreach*.

<img width="1093" alt="image" src="https://github.com/OSGeo/osgeo/assets/629681/1eb7f353-fe23-495f-9943-e30565477250">

I am marking this as *done*, but consider it poorly done...

**Comment by jodygarnett** [comment]
I am re-opening this issue, and restoring the "About OSGeo" button based on feedback on [discourse](https://discourse.osgeo.org/t/change-website-text-on-free-software-open-source-foss4g/28205/4). 

This is simply the wrong location for a donate button and changes the whole tone of the website and relationship osgeo offers from "here is what we offer you" to "how can you help us". It is literally the first thing we ask people to do - not cool :)

The "About OSGeo" button was part of the original design and without this information there action to find out about the foundation and who we are and what we offer. 

<img width="1042" alt="image" src="https://github.com/OSGeo/osgeo/assets/629681/fa41e992-7991-425f-9982-024de64a0164">

**Comment by jodygarnett** [comment]
Website is restored - if we do want a donate button it needs to be part of the visual design and part of the story. Also PayPal is not our preferred method of accepting donations/sponsorships.

examples: 

- if we highlighted a sponsor we could have buttons for "Sponsor" and "Donate".
- Support our community heading with "Sponsor" and "Donate" buttons
